src/app/components/nav-bar/nav-bar.component.html

```diff
@@ -0,0 +1,22 @@
+<nav class="navBar">
+  <div class="link" routerLink="/heroes-hall" routerLinkActive="active">
+    <p>Heroes hall</p>
+  </div>
+  <div class="menu link" *ngIf="isSelected()">
+    <div class="subLink">
+      <p>{{heroName}}</p>
+    </div>
+    <div class="subLink" (click)="navToPage('character')">
+      <p>Character</p>
+    </div>
+    <div class="subLink" (click)="navToPage('inventory')">
+      <p>Inventory</p>
+    </div>
+    <div class="subLink" (click)="navToPage('dungeon')">
+      <p>Dungeon</p>
+    </div>
+  </div>
+  <div class="logout link" routerLink="/login" routerLinkActive="active" (click)="logout()">
+    <p>Logout</p>
+  </div>
+</nav>
```

Comment:
we have to discuss the navbar to make it bootstrap available

Reply:
lets make that into a new jira task